kernel/inductive.ml

```diff
@@ -126,7 +126,7 @@ let max_template_universe u v = u @ v
 (* to [x]. *)
 let cons_subst u su subst =
   let su = match su with
-  | Sorts.SProp -> assert false (* No template on SProp *)
+  | Sorts.SProp -> [] (* pretend it's Prop, will fail later *)
```

Comment:
I think this is too dangerous a fix. You should prevent this from happening somewhere upstream instead.